charts/huawei-csi-plugin/templates/_helpers.tpl

```diff
@@ -0,0 +1,62 @@
+{{/*
```

Comment:
We could use `charts.adfinis.com/common` instead of rewriting `_helpers.tpl` file every time. In fact we should also add some `selectorLabels` functionality in the common chart.

Reply:
I've created an [issue](https://github.com/adfinis-sygroup/helm-charts/issues/182) as discussed to cover this request in future.